dexbot/helper.py

```diff
@@ -57,3 +58,26 @@ def initialize_orders_log():
 
     if not file:
         logger.info("worker_name;ID;operation_type;base_asset;base_amount;quote_asset;quote_amount;timestamp")
+
```

Comment:
Doing the try/except test inside the find_external_strategies function would make this a bit more readable, don't you think?